defaults/validate.go

```diff
@@ -94,6 +96,22 @@ func Validate(value interface{}) error {
 }
 
 func validate(validation, fieldName string, p, v reflect.Value) error {
+	// Multiple validations separated by a colon (as in "empty;file") behave like an OR clause
+	if strings.Contains(validation, ";") {
+		individualValidations := strings.Split(validation, ";")
+		merr := merror.New()
+		for _, validation := range individualValidations {
+			err := validate(validation, fieldName, p, v)
+			// Return as soon as one is valid
+			if err == nil {
+				return nil
+			}
+
+			merr.Append(err)
```

Comment:
You can just use the standard library: https://pkg.go.dev/errors#Join

Reply:
Oh, didn't know about that one, thanks!